src/loc/dashboard/WorkInProgressLocs.tsx

```diff
@@ -0,0 +1,76 @@
+import { LocData, LocRequestState } from "@logion/client";
+import { LocType, UUID } from "@logion/node-api";
+import { useMemo } from "react";
+import { useNavigate } from "react-router-dom";
+import Button from "src/common/Button";
+import ButtonGroup from "src/common/ButtonGroup";
+import { useCommonContext } from "src/common/CommonContext";
+import LegalOfficerName from "src/common/LegalOfficerNameCell";
+import LocStatusCell from "src/common/LocStatusCell";
+import Table, { Cell, Column, DateTimeCell, EmptyTableMessage } from "src/common/Table";
+import UserIdentityNameCell from "src/common/UserIdentityNameCell";
+
+export interface Props {
+    locs: LocRequestState[];
+    locDetailsPath: (locId: UUID, locType: LocType) => string;
+}
+
+export default function WorkInProgressLocs(props: Props) {
```

Comment:
This table replaces, among others, the table `RejectedLocRequests` which displayed the reject reason. Is it still viewable somewhere ?

Reply:
I forgot about that, bringing it back.